src/Microsoft.OData.Core/JsonLight/ODataJsonLightOutputContext.cs

```diff
@@ -132,7 +132,8 @@ internal sealed class ODataJsonLightOutputContext : ODataOutputContext
             Debug.Assert(messageWriterSettings != null, "messageWriterSettings != null");
 
             this.textWriter = textWriter;
-            this.jsonWriter = CreateJsonWriter(messageInfo.Container, textWriter, true /*isIeee754Compatible*/, messageWriterSettings);
+            bool ieee754CompatibleSetToTrue = (messageInfo.MediaType != null) ? messageInfo.MediaType.HasIeee754CompatibleSetToTrue() : false;

```

Comment:
I cant see where `HasIeee754CompatibleSetToTrue` is defined. It doesn't exist in MediaType

Reply:
It is an ODataMediaType extension method defined in [MediaTypeUtils.cs ](https://github.com/paulodero/odata.net/blob/31573526cf9a9359c35f101875fd2f39f5341b26/src/Microsoft.OData.Core/MediaTypeUtils.cs#L324)class